debian/rtw89-dkms.dkms

```diff
@@ -1,14 +1,23 @@
-PACKAGE_VERSION="@@VERSION@@"
 PACKAGE_NAME="rtw89"
-PACKAGE_VERSION="1.0.1"
+PACKAGE_VERSION="#MODULE_VERSION#"
 MAKE="make -C ${kernel_source_dir} KBUILD_EXTMOD=${dkms_tree}/${PACKAGE_NAME}/${PACKAGE_VERSION}/build modules"
 CLEAN="make -C ${kernel_source_dir} KBUILD_EXTMOD=${dkms_tree}/${PACKAGE_NAME}/${PACKAGE_VERSION}/build clean"
 BUILT_MODULE_NAME[0]="rtw89pci"
 BUILT_MODULE_NAME[1]="rtw89core"
 BUILT_MODULE_NAME[2]="rtw_8852a"
 BUILT_MODULE_NAME[3]="rtw_8852ae"
+BUILT_MODULE_NAME[4]="rtw_8852b"
+BUILT_MODULE_NAME[5]="rtw_8852be"
+BUILT_MODULE_NAME[6]="rtw_8852c"
+BUILT_MODULE_NAME[7]="rtw_8852ce"
 DEST_MODULE_LOCATION[0]="/updates/"
 DEST_MODULE_LOCATION[1]="/updates/"
 DEST_MODULE_LOCATION[2]="/updates/"
 DEST_MODULE_LOCATION[3]="/updates/"
+DEST_MODULE_LOCATION[4]="/updates/"
+DEST_MODULE_LOCATION[5]="/updates/"
+DEST_MODULE_LOCATION[6]="/updates/"
+DEST_MODULE_LOCATION[7]="/updates/"
 AUTOINSTALL="YES"
+# Set the acceptable kernel values - 5.7+
+BUILD_EXCLUSIVE_KERNEL="^(5\.0[7-9]+\.|5\.[1-9][0-9]+\.|6.[0-9]+\.)"
```

Comment:
OK - I see the error here - will re-push

Reply:
I tested this against ubuntu focal (5.4) - but I didn't have a 5.8+ kernel around - the old one would exclude all 5.[0-9] versions, this one should allow 5.[7-9]